docs/proposals/201812_thanos-remote-receive.md

```diff
@@ -106,23 +106,40 @@ Hard tenants in the Thanos receiver are configured in a configuration file. Chan
 
 A sample of the configuration of tenants and their respective infrastructure:
 
-```json
-[
-    {
-        "hashring": "tenant-a",
-        "endpoints": ["tenant-a-1.metrics.local:19291/api/v1/receive", "tenant-a-2.metrics.local:19291/api/v1/receive"],
-        "tenants": ["tenant-a"]
-    },
-    {
-        "hashring": "tenants-b-c",
-        "endpoints": ["tenant-b-c-1.metrics.local:19291/api/v1/receive", "tenant-b-c-2.metrics.local:19291/api/v1/receive"],
-        "tenants": ["tenant-b", "tenant-c"]
-    },
-    {
-        "hashring": "soft-tenants",
-        "endpoints": ["http://soft-tenants-1.metrics.local:19291/api/v1/receive"]
-    }
-]
+```yaml
+route:
+- hashring_group: soft-tenants
```

Comment:
I don't see an example config for a hard tenant?

Reply:
The other hashring groups are hard tenants, notice that this one has no `match` section`, which means it is a catch-all. Also note that the hashring group name `soft-tenants` has no magic properties

Reply:
aah , then the identation is wrong, no? It seems like the other `hashring_group` are under `hashring_group: soft-tenants`

Reply:
not quite, the `hashring_groups` field is for defining a list of all hashrings. The `route` tree is for assigning hashring groups to routes

Reply:
aah I get it now, but some commenting would be nice.